app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -165,28 +187,32 @@ object MarkdownHelper {
         color: Color = Color.Unspecified,
         onClick: (() -> Unit)? = null,
         onLongClick: (() -> Unit)? = null,
+        maxLines: Int? = null,
+        style: TextStyle? = null,
+        includeImages: Boolean = true,
     ) {
-        val style = MaterialTheme.typography.bodyLarge
         val defaultColor: Color = LocalContentColor.current.copy(alpha = LocalContentAlpha.current)
-
+        val typographyStyle = style ?: MaterialTheme.typography.bodyLarge
```

Comment:
Considering this is a default, move this into the params instead.

Reply:
updated